search/Property.cc

```diff
@@ -957,7 +960,8 @@ getProperty(const Instance *inst,
     return PropertyValue(liberty_cell && liberty_cell->isInverter());
   else if (stringEqual(property, "is_macro"))
     return PropertyValue(liberty_cell && liberty_cell->isMacro());
-  else if (stringEqual(property, "is_memory_cell"))
+  else if (stringEqual(property, "is_memory")
+	   || stringEqual(property, "is_memory_cell"))
```

Comment:
I am not sure why the original code used "is_memory_cell" but I think we should just flush it unless you see a use for it.

Reply:
Ok